<launch>

<!-- Exploration Node -->
<node name="exploration" pkg="exploration" type="exploration" output="screen" launch-prefix="xterm -e" >
</node>

<node name="dynamic_reconfigure" pkg="dynamic_reconfigure" type="reconfigure_gui" output="screen">
</node>



</launch>
